uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/controllers/sub-controllers/request-modal-details-toil.controller.spec.js

```diff
@@ -170,6 +174,17 @@ define([
           expect(controller.request.toil_duration).toBe(controller.uiOptions.toil_duration_in_hours * 60);
         });
 
+        describe('when "to" date selected that is greater than "from" date', function () {
+          beforeEach(function () {
+            requestModalHelper.setTestDates(controller, '03/22/2018');
```

Comment:
`03/22/2018` `03/22` ? since when do we use fairy date formats?

Also is `22/03/2018` the TO date? what is the FROM date?

Do you think it would be possible to do something like

```js
var fromDate = moment().format('format here');
var toDate = moment().add(1, 'week').format('format here');

// pass the date to the setTestDates method
```